### PR TITLE
[dev-v2.6] February RKE2 patches

### PR DESCRIPTION
**Pull request by brooksn**

### CODE REVIEW

**Review by rancher-max — CHANGES_REQUESTED** [review]
I think the branch is incorrect -- should be `dev-v2.6-0123-patches` unless something changed @snasovich @kinarashah ?

**Comment by kinarashah** [comment]
@rancher-max  The latest I have on feb patches is 

**Feb patches:**
2.6 target branch: `dev-v2.6-for-2.7.2`
**Todo**: Will create branch and sync changes from `dev-v2.6` to `dev-v2.6-for-2.7.2` after `dev-v2.6` is up to date
2.7 target branch: `dev-v2.7`

I am yet to finish up the todo item and sync branches, we released KDM today so I'll be able to get this done tomorrow.

**Review by rancher-max — DISMISSED** [review]
I know in the past we made maxChannelServerVersion 2.6.99, but I like it as 2.7.99 so I'm approving this 😄

File: channels-rke2.yaml [hunk 1]
```diff
@@ -890,6 +890,26 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1500
     featureVersions: *featureVersions-v1
+  - version: v1.23.17+rke2r1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.7.99
```
Comment (snasovich) [inline channels-rke2.yaml]: We've been doing `maxChannelServerVersion: v2.6.99` for 2.6 releases except of `v1.23.16+rke2r1` and `v1.24.10+rke2r1` where it was accidentally set to `v2.7.99`. I think we want to go back to setting it to `v2.6.99` for consistency.

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1033,4 +1053,24 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1500
     featureVersions: *featureVersions-v1
+  - version: v1.24.11+rke2r1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.7.99
```
Comment (snasovich) [inline channels-rke2.yaml]: Same as the other comment regarding `v2.7.99` vs. `v2.6.99`.

**Review by rancher-max — APPROVED** [review]

**Pull request merged**